src/components/ProfileValidationSteps/VerifyIdentify.tsx

```diff
@@ -34,7 +37,13 @@ export const VerifyIdentify = (props: VerifyIdentifyProps) => {
           <Typography variant='headline1'>In order to validate your identify, we require:</Typography>
             <Typography variant='body1'>1. Accounts to have an ORCID. Please link your profile below. </Typography>
             <div className='orcid-container'>
-              {verificationSubmission.orcid && <Typography variant='body1'><img className='verifyBadgeIcon' src={CheckmarkBadge} alt='CheckmarkBadge'/>ORCiD: {verificationSubmission.orcid}</Typography>}
+              {
+              verificationSubmission.orcid &&
+              <>
+                <Typography style={{display:'inline-block', marginRight:'16px'}} variant='body1'><img className='verifyBadgeIcon' src={CheckmarkBadge} alt='CheckmarkBadge'/>ORCiD: {verificationSubmission.orcid}</Typography>
+                <Button onClick={()=>setShowORCiDDialog(true)} style={{minWidth:'150px'}} className='btn-container emptyButton'>Edit Link</Button>
```

Comment:
I think this Edit also only makes sense if an ORCiD has been bound.
So I believe this should show if !verificationSubmission.orcid, and should use the edit icon (instead of the text "Edit Link")

Reply:
Currently the edit button shows when the ORCiD is bound. When it isn't the Link My ORCiD button is shown. As for the "Edit Link" button, it was what the figma had but can easily be changed.